templates/pubspecs/tools/serverpod_cli/pubspec.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   http: ^0.13.3
   intl: ^0.17.0
   recase: ^4.0.0
+  built_collection: ^5.1.1
```

Comment:
Is this neccessary to add? I cannot see that it is used anywhere.

Reply:
This library needs to be used for 
```dart 
classBuilder.implements = ListBuilder([refer('ServerpodException', serverpodUrl(serverCode))]);
```
Because  **_`ListBuilder`_** is not visible without this import.

Reply:
Ok cool.